js/loader.js

```diff
@@ -35,7 +35,7 @@ var Loader = (function() {
 				// This is done after all the moduels so we can
 				// overwrite all the defined styls.
 
-				loadFile("css/custom.css", function() {
+				loadFile(config.customCss, function() {
```

Comment:
If set a wrong path for this configuration variable all system is crashed. It's to fragile ...I thing can be a handler in case of error. 

So, this can be refactored to add a test case